apps/bfd-data-fda/src/main/java/gov/cms/bfd/data/fda/lookup/FdaDrugCodeDisplayLookup.java

```diff
@@ -43,62 +37,24 @@ public class FdaDrugCodeDisplayLookup {
   /** Keeps track of the SUBSTANCENAME column index in the fda_products_utf8.tsv file. */
   private static int SUBSTANCE_NAME_COLUMN_INDEX = 13;
 
-  /**
-   * Cached copy of the testing version of the {@link FdaDrugCodeDisplayLookup} so that we don't
-   * have to construct it over and over in the unit tests.
-   */
-  private static FdaDrugCodeDisplayLookup drugCodeLookupForTesting;
-
   /**
    * Cached copy of the production version of the {@link FdaDrugCodeDisplayLookup} so that we don't
    * have to construct it over and over in the unit tests.
    */
   private static FdaDrugCodeDisplayLookup drugCodeLookupForProduction;
 
-  /**
-   * Factory method for creating a {@link FdaDrugCodeDisplayLookup} for testing that includes the
-   * fake drug code.
-   *
-   * @return the {@link FdaDrugCodeDisplayLookup}
-   */
-  public static FdaDrugCodeDisplayLookup createDrugCodeLookupForTesting() {
-    if (drugCodeLookupForTesting == null) {
-      drugCodeLookupForTesting = new FdaDrugCodeDisplayLookup(true);
-    }
-
-    return drugCodeLookupForTesting;
-  }
-
-  /**
-   * Factory method for creating a {@link FdaDrugCodeDisplayLookup} for production that does not
-   * include the fake drug code.
-   *
-   * @return the {@link FdaDrugCodeDisplayLookup}
-   */
-  public static FdaDrugCodeDisplayLookup createDrugCodeLookupForProduction() {
-    if (drugCodeLookupForProduction == null) {
-      drugCodeLookupForProduction = new FdaDrugCodeDisplayLookup(false);
-    }
-
-    return drugCodeLookupForProduction;
-  }
-
-  /**
-   * Constructs a {@link FdaDrugCodeDisplayLookup} for testing purposes only.
-   *
-   * @param ndcProdMap Drug code lookup map to populate FdaDrugCodeDisplayLookup
-   */
-  public FdaDrugCodeDisplayLookup(Map<String, String> ndcProdMap) {
-    ndcProductHashMap.putAll(ndcProdMap);
+  /** Constructs an {@link FdaDrugCodeDisplayLookup}. */
+  public FdaDrugCodeDisplayLookup() {
```

Comment:
I think this constructor is no longer used now that the other one always accepts the filename